contracts/exchange-forwarder/contracts/src/libs/LibConstants.sol

```diff
@@ -27,28 +27,26 @@ contract LibConstants {
 
     using LibBytes for bytes;
 
-    bytes4 constant internal ERC20_DATA_ID = bytes4(keccak256("ERC20Token(address)"));
-    bytes4 constant internal ERC721_DATA_ID = bytes4(keccak256("ERC721Token(address,uint256)"));
+    bytes4 constant internal ERC20_PROXY_ID = bytes4(keccak256("ERC20Token(address)"));
```

Comment:
We can probably remove these in favor of the `IAssetData(address(0)).ERC20Token.selector` pattern using https://github.com/0xProject/0x-monorepo/blob/development/contracts/asset-proxy/contracts/src/interfaces/IAssetData.sol.